### PR TITLE
Using pytest with Django

### PR DESCRIPTION
**Pull request by cameronmaske**
Todos:

- [x] Upload example repo
- [x] Add readme to example repo
- [x] Check all links. 

This should be ready for a draft review @joshfinnie 

I also have a few commits to improve the build/update my author bio mixed in here.

### CODE REVIEW

**Comment by cameronmaske** [comment]
@joshfinnie Example repo up and running! Just pending review.

File: content/using-pytest-with-django.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        self.assertEqual("hello world", "hello world")
+```
+
+#### `pytest`
+```
+def test_hello_world():
+    assert "hello_world" == "hello_world"
+```
+
+In our newer Django projects we've been using `pytest` instead of the default test runner. Django's prefer testing is built on top of [unittest](https://github.com/django/django/blob/2ab244ff3a799b1a49550a7e7582c4b46e402197/django/test/testcases.py#L155). Luckily, for any `pytest` wannabes this makes it easy to have `pytest` be a drop in [replacement](https://pytest.org/latest/unittest.html) without having to change a single test.
```
Comment (joshfinnie) [inline content/using-pytest-with-django.md]: The use of `wannabes` is weird here... maybe "fans"?

File: content/using-pytest-with-django.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+# We manually designate which settings we will be using in an environment variable
+# This is similar to what occurs in the `manage.py`
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'app.config.settings')
+
+
+# `pytest` automatically calls this function once when tests are run.
+def pytest_configure():
+    settings.DEBUG = False
+    # If you have any test specific settings, you can declare them here,
```
Comment (joshfinnie) [inline content/using-pytest-with-django.md]: Do you think removing the commented-out lines here would help clarify what's actually going on in this function?

File: theme/templates/bios/cameron-maske.html [hunk 3]
```diff
@@ -1,2 +1,2 @@
-<p class="well-content">I am a Senior Software Maven at <a href="http://trackmaven.com/">TrackMaven</a>.</p>
+<p class="well-content">I am the Lead Architect Maven at <a href="http://trackmaven.com/">TrackMaven</a>.</p>
```
Comment (joshfinnie) [inline theme/templates/bios/cameron-maske.html]: Woot!

**Comment by joshfinnie** [comment]
@cameronmaske made a few comments above, let me know what you think...

File: content/using-pytest-with-django.md (hunk 1 above)
Comment (cameronmaske) [inline content/using-pytest-with-django.md]: I agree!

File: content/using-pytest-with-django.md (hunk 2 above)
Comment (cameronmaske) [inline content/using-pytest-with-django.md]: I don't think so, I'm kind of using it guide through the code. I think it wouldn't be clear without them.

**Comment by joshfinnie** [comment]
@cameronmaske wanna make that one change and then rebase? I think we should post this tomorrow!

If you agree, wanna update the date as well before rebasing?

**Comment by cameronmaske** [comment]
@joshfinnie Done!

**Comment by cameronmaske** [comment]
@joshfinnie Date should be updated now!

**Pull request merged**